.github/workflows/build-test-deploy.yml

```diff
@@ -298,6 +296,10 @@ jobs:
     runs-on: ubuntu-latest
     needs: [ test, build_and_push ]
     environment: stage-manual
+    permissions:
+      id-token: write
+      contents: read
+      packages: read
```

Comment:
Reading up on this. What are the default permissions we use? Can we set them to 'restricted' ?